site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -889,9 +889,8 @@ components:
                   `{{subject-ou}}` and `{{subject-o}}` are currently supported. The template MUST contain at least
                   one of these placeholders. The `device-id` is generated by replacing any placeholders in the template
                   string with their corresponding *Common Name (CN)*, *Subject DN (DN)*, *Organizational Unit Name (OU)*
-                  and *Organization Name (O)* from the client certificate used for authentication by unregistered
-                  devices and gateways. The *Subject DN* is formatted according to the rules defined in
-                  [RFC 2253](https://tools.ietf.org/html/rfc2253#section-2).
+                  and *Organization Name (O)* from the client certificate's subject DN. The *Subject DN* is formatted
```

Comment:
the subject DN from the certificate's subject DN?

Reply:
I have corrected it now and pushed a commit. Would you mind taking another look?